python-client/giskard/models/model_explanation.py

```diff
@@ -47,7 +104,24 @@ def _prepare_for_explanation(input_df: pd.DataFrame, model: BaseModel, dataset:
 
 
 def _calculate_dataset_shap_values(model: BaseModel, dataset: Dataset) -> np.ndarray:
-    """Perform SHAP values calculation for samples of a given dataset."""
+    """Perform Kernel SHAP explanation of the `dataset` entries.
+
+    Return SHAP contributions of features from the `dataset` to the output of
```

Comment:
Returns the feature importance in terms of SHAP values.

Reply:
\+